packages/blockchain-link-utils/src/solana.ts

```diff
@@ -375,7 +376,22 @@ export const getTxType = (
         return 'failed';
     }
 
-    // we consider only parsed instructions because only based on them we can determine the type of transaction
+    const isThirdPartyProgramInstruction = (
```

Comment:
I'd call this `isUnknownProgramInstruction` since for now we also don't support e.g. StakeProgram, which isn't really a "third-party" program.